platforms/substrate/configuration/roles/create/genesis/tasks/main.yaml

```diff
@@ -0,0 +1,73 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# This task creates a build directory within the directory that this play is in
+- name: Create build directory if it does not exist
+  file:
+    path: "./roles/create/genesis/build"
+    state: directory
+
+# This task checks if genesis is already in the vault for each org
+- name: Check if genesis is in the vault for each org
+  include_tasks: check_vault.yaml
+  vars:
+    component_ns: "{{ org.name | lower }}-subs"
+    vault: "{{ org.vault }}"
+  loop: "{{ network['organizations'] }}"
+  loop_control:
+    loop_var: org
+
+# This task create the template genesis file and stores it in the build folder
+- name: Create the base genesis file
+  shell: |
+    CONTAINER="{{ network.docker.url }}/{{ network.config.node_image }}:{{ network.version }}"
+    docker pull $CONTAINER
+    BASE_CHAIN="local"
+    GENESIS_DIR="$(pwd)/roles/create/genesis/build"
+    
+    GENESIS=$(docker run --rm -a stdout $CONTAINER build-spec --disable-default-bootnode --chain $BASE_CHAIN)
+    GENESIS=$(echo $GENESIS | jq '.chainType |= "Live"')
+    GENESIS=$(echo $GENESIS | jq '.genesis.runtime.palletAura.authorities |= []')
+    GENESIS=$(echo $GENESIS | jq '.genesis.runtime.palletGrandpa.authorities |= []')
```

Comment:
This task should be a job as we cannot be sure that docker is available on ansible machine. The job should store the generated genesis to Vault, then ansible will download it to add the grandpa and aura info, and push it back and store in the path provided by network.config.genesis